app/views/jobs/getjob.html.erb

```diff
@@ -31,10 +31,10 @@
 </tr></table>
 
 <% if @cud.user.administrator?  then %>
-<li><kbd>notifyURL</kbd>: <kbd><%= @job[:rjob]["notifyURL"] %> </kbd>
+<li><kbd>notifyURL</kbd>: <%= @job[:rjob]["notifyURL"] %>
```

Comment:
Why couldn't this be changed to normal English? It just bugs me that one of five is special like this, but it's not a huge issue.